README.md

```diff
@@ -1 +1,103 @@
+   
+   If you can't find a PR for your plugin, you can manually trigger one as explained below.
+   
+   ### Create a PR using the "Update plugins repository references" workflow
+
+  > [!IMPORTANT]
+  > Write access to this repository is required to run this workflow.
+
+   - Navigate to https://github.com/redhat-developer/rhdh-plugin-export-overlays/actions/workflows/update-plugins-repo-refs.yaml 
+   - For "use workflow from" select main
+   - For "regexps", specify the regular expression matching the plugins you want to package. For example, to package all RBAC plugins, the regexp would be "@backstage-community/plugin-rbac"
+   - Running the workflow with the following inputs should generate a PR similar to [these ones](https://github.com/redhat-developer/rhdh-plugin-export-overlays/pulls/app%2Fgithub-actions)
+    
+   **How it works**:
+   - The workflow generates PRs against each active release branch corresponding to an RHDH release.
+   - For each release branch, it checks for plugin versions compatible with the Backstage version that the release supports.
+   - If no compatible version is found, no PR is generated for that plugin on that branch.
+
+### Manually Creating a PR
+
+You can also create PRs manually by referencing existing examples: [View PR examples](https://github.com/redhat-developer/rhdh-plugin-export-overlays/pulls/app%2Fgithub-actions)
+
+To add a new plugin:
+
+1. Create a new workspace in the overlay repository.
+2. Add a `plugins-list.yaml` file that lists all plugins included in the target workspace of the source repository. ([See example](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/12acb71a1febc5567c4d12c6a28c0a11ed489273/workspaces/adoption-insights/plugins-list.yaml))
+3. Add a `source.json` file with the following fields ([See example](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/12acb71a1febc5567c4d12c6a28c0a11ed489273/workspaces/adoption-insights/source.json))
+:
+   - `repo`: URL of the source repository  (only `https://github.com/xxx` URLs are supported for now)
+   - `repo-ref`: Specific tag or commit for the target plugin/workspace version  
+   - `repo-flat`:  
+     - `false` if the plugins are inside a workspace (e.g., `backstage/community-plugins`)  
+     - `true` if the plugins are at the root level (e.g., `backstage/backstage`)
+
+### 3. Add Additional Dynamic Plugin Export Information (If Needed)
+
+Sometimes, additional configuration is required in the PR:
+
+- **Frontend plugins** may need:
+   - `app-config.dynamic.yaml` (Eg: [techdocs plugin](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/release-1.5/workspaces/backstage/plugins/techdocs/app-config.dynamic.yaml))
+   - `scalprum-config.json` (Eg: [api-docs-module-protoc-gen-doc plugin](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/release-1.5/workspaces/backstage/plugins/api-docs-module-protoc-gen-doc/scalprum-config.json))
```

Comment:
In most case this one should not be necessary, especially due to the fact that, in the future, we would move away from Scalprum.
Not sure we should even mention this here.

Reply:
Or maybe just mention that this is only rarely required ?